master/contrib/hgbuildbot.py

```diff
@@ -156,24 +246,24 @@ def _printFailure(why):
     d.addBoth(lambda _: reactor.stop())
     reactor.run()
 
-    if fork:
-        os._exit(os.EX_OK)
-    else:
-        return
-
-# taken from the mercurial notify extension
-
 
 def strip(path, count):
-    '''Strip the count first slash of the path'''
-
+    """Strip the count first slash of the path"""
     # First normalize it
     path = '/'.join(path.split(os.sep))
-    # and strip it part after part
-    while count > 0:
-        c = path.find('/')
-        if c == -1:
-            break
-        path = path[c + 1:]
-        count -= 1
-    return path
+    # and strip the *count* first slash
+    return path.split('/', count)[-1]
+
+
+def send_cs(res, s, c):
```

Comment:
We try to avoid this kind of variable names: `s` and `c`.  They are not descriptive.

If you'd replace them with `sender` and `changeset` respectively, the doc string is redundant.